myconext-server/src/main/java/myconext/security/GuestIdpAuthenticationRequestFilter.java

```diff
@@ -719,7 +719,7 @@ protected List<SAMLAttribute> attributes(User user, String requesterEntityId) {
         List<SAMLAttribute> attributes = new ArrayList(Arrays.asList(
                 attribute("urn:mace:dir:attribute-def:cn", displayName),
                 attribute("urn:mace:dir:attribute-def:displayName", displayName),
-                attribute("urn:mace:dir:attribute-def:commonName", commonName),
+                attribute("urn:mace:dir:attribute-def:cn", commonName),
```

Comment:
This is also done on line 720 but then mapped to displayName, so might not do what you expect.